Classes/Report/ServicesListReport.php

```diff
@@ -184,7 +184,7 @@ protected function groupFileTypes(array $fileTypes)
      *
      * @return \TYPO3\CMS\Lang\LanguageService
      */
-    protected function getLanguageService()
+    protected function getLanguageService(): \TYPO3\CMS\Lang\LanguageService
```

Comment:
LanguageService is part of core now. Not lang.

Reply:
So what do you expect here?

Reply:
\TYPO3\CMS\Core\LanguageService

Reply:
done - thought it just arrived with v10 ...

Reply:
zumindest phpstan ist hier auch der meinung, dass das nicht passt